scripts/export_helix_candidate_profile.py

```
@@ -0,0 +1,198 @@
+        item = _clean(value)
+        key = item.casefold()
+        if item and key not in seen:
+            seen.add(key)
+            result.append(item)
+    return result
+
+
+def _period(entry: dict[str, Any]) -> str:
+    start = _clean(entry.get("startDate"))
+    end = _clean(entry.get("endDate")) or "present"
+    return "–".join(part for part in (start, end) if part)
```

Ruff (0.16.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)